content/momentum/mobile/index.md

```diff
@@ -12,6 +14,4 @@ Confidential & Proprietary.
 
 **Abstract**
 
-This book is an administration guide for Mobile Momentum.
-
----
+This book is an administration guide for Mobile Momentum.
```

Comment:
This change is undoing some of the changes made here: https://github.com/SparkPost/support-docs/pull/612/files